packaging/macos/Portfile

```diff
@@ -32,6 +32,7 @@ post-fetch {
 }
 
 depends_lib              port:avahi \
+                         port:boost180 \
```

Comment:
I don't think this is needed due to the `boost.version` property below.

Reply:
The boost port is just a shim around 1.76, I couldn't get it to link without explicitly choosing this. https://ports.macports.org/port/boost/details/